Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -225,6 +236,7 @@ function Get-ExchangeInformation {
             ExtendedProtectionConfig                 = $extendedProtectionConfig
             ExchangeConnectors                       = $exchangeConnectors
             ExchangeServicesNotRunning               = [array]$exchangeServicesNotRunning
+            TransportService                         = $transportService
```

Comment:
Let's also use `GetTransportService` as that should be what the other `Get-*` cmdlets that we store the whole object are using.